Python/Calculator/calculator.py

```diff
@@ -0,0 +1,85 @@
+'''
+INSTRUCTIONS
+
+Press RUN
+Enter first number
+click enter button on the keyboard
+enter operator
+click enter button again
+enter the second number
+click SUBMIT button
+
+
+FOR OPERATORS ENTER THE FOLLOWING
+
+you can use upper or lower case for operations    
+   +     for addition
+   -     for subtraction
+   /     for division
+   *     for multiplication
+   ^     for power
+   r     for root
+   %     for modulus
+   pie   for Pie
+   e     for E
+   sin   for sin (trig)
+   cos   for cos (trig)
+   tan   for tan (trig)
+   !     for factorial 
+   ln    for ln (natural log)
+   
+error? read comment below
+
+for PI and E you need to put any numbers just to prevent input error but write PI or E for operator.
+
+for FACTORIAL and TRIG you need to put zero (or any number) as the first number, then the sign and last the number you want to calculate. The first number (or the zero) will not be used for calculations but it will prevent an error
+
+
+THE CODE
+'''
+import math
+from builtins import print
+
+print ("Please read instructions in the code\nor reach https://github.com/kaustubhgupta/Hacktoberfest2020_ for help \n*************************")
+firstNumber = float(input())
+op = input().lower()
+secondNumber = float(input())
+
+# functions of the calculator
+if op == "+":
```

Comment:
Implement a class-based code, move all this code to a class, and create separate functions for each operation. If the operator requires only one operand then don't take the second number input and directly output the result.

Reply:
What about the class code?